package.json

```diff
@@ -10,9 +10,7 @@
   "scripts": {
     "start": "nodemon ./server/server.js",
     "test": "jest",
-    "build": "webpack --config webpack.config.js",
-    "maria": "mariadb <'./server/database/schema.sql';",
-    "seed": "node ./server/database/dbSeeder.js"
+    "build": "webpack --config webpack.config.js"
```

Comment:
I think you need to add a --watch to your build config so that it is always watching. I remember one of the TMs telling us that.